src/models/comment.js

```diff
@@ -0,0 +1,22 @@
+//Requires Mongoose
+const mongoose = require("mongoose");
+
+//Creates variables to store Schema
+
+const Schema = mongoose.Schema;
+
+//Creates comment schema
+
+const commentSchema = Schema({
+  userID: { type: Schema.Types.ObjectId, ref: "User" },
```

Comment:
This should be id or _id? Id is created by mongo?

Reply:
_id is automatically will be generated by the mongoDB as soon as you save the data, you don't need to set up inside the schema.